airbyte-ci/connectors/pipelines/pyproject.toml

```diff
@@ -20,8 +20,8 @@ semver = "^3"
 airbyte-protocol-models = "*"
 jinja2 = "^3.0.2"
 requests = "^2.31"
-airbyte-connectors-base-images = {path = "../base_images", develop = true}
-connector-ops = {path = "../connector_ops", develop = true}
+airbyte-connectors-base-images = { path = "../base_images", develop = true }
+connector-ops = { path = "../connector_ops", develop = true }
 toml = "^0.10.2"
 types-requests = "^2.31"
 sentry-sdk = "^1.28.1"
```

Comment:
Should've upgraded Sentry too tbh. Felt lazy, I will merge this on green CI and upgrade Sentry separately.